src/pages/Dashboard/scenes/Emails/Emails.tsx

```diff
@@ -1,9 +1,224 @@
+                          onChange={(
+                            e: React.ChangeEvent<HTMLTextAreaElement>
+                          ) => {
+                            setSelectedMentees(e.target.value.split(','));
+                          }}
+                          placeholder="Enter recipient emails, separated by commas"
+                          className="mt-1 px-4 py-2 block w-full rounded-md border-gray-300 shadow-sm"
+                        />
+                      </label>
+                      <label className="block">
+                        <span className="text-gray-700">Body:</span>
+                        <textarea
+                          value={body}
+                          placeholder="Write body message here"
+                          onChange={(e) => {
+                            setBody(e.target.value);
+                          }}
+                          className="mt-2 px-4 py-4 block w-full rounded-md border-gray-300 shadow-sm"
+                        />
+                      </label>
+                      <div>
+                        {isLoading ? (
+                          <button
+                            type="submit"
+                            className="mt-4 x-6 py-2 bg-blue-500 text-white rounded hover:bg-blue-700 cursor-pointer"
+                          >
+                            <div className="flex justify-center items-center h-5 px-6 py-2">
+                              <>
+                                <LoadingSmallSVG />
+                              </>
+                            </div>
+                          </button>
+                        ) : (
+                          <button
+                            type="submit"
+                            className="mt-4 px-6 py-2 bg-blue-500 text-white rounded hover:bg-blue-700 cursor-pointer"
+                          >
+                            Send
+                          </button>
+                        )}
+                        {message.length > 0 && (
+                          <div className="bg-blue-100 border-t-4 border-blue-500 rounded-b text-blue-900 px-4 py-3 shadow-md my-3">
+                            {message}
+                          </div>
+                        )}
+                      </div>
+                    </div>
+                  </form>
+                </div>
+              </div>
+              <div className="bg-gray-100 p-6 rounded shadow-lg">
+                <h2 className="text-xl font-bold mb-2 text-blue-500">
+                  Email Preview here
+                </h2>
+                <div className="bg-white p-4 rounded shadow">
+                  <EmailTemplate
+                    subject={subject}
+                    body={body}
+                    recipient={select}
+                  />
+                </div>
+              </div>
+            </div>
+          </>
+        ) : (
+          <>
+            <EmailHistory />
+          </>
```

Comment:
```suggestion
            <EmailHistory />
```

Reply:
@anjula-sack  in history component, I'm just rendering each email's sender email, subject, status of sent,  open status, and sent time. so when I sent test emails with long subject text, that table quite getting big row each emails. 
![image](https://github.com/sef-global/scholarx-frontend/assets/48531182/5863451c-3c52-41e6-8293-23ca26ba4b5d)

Reply:
Truncate the subject using taiilwind css

Reply:
okay I'll do it